jicoco-test-kotlin/src/main/kotlin/org/jitsi/config/LegacyFallbackConfigTester.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.config
+
+import io.kotlintest.matchers.withClue
+import io.kotlintest.shouldBe
+import org.jitsi.utils.config.ConfigProperty
+import org.jitsi.videobridge.testutils.EMPTY_CONFIG
```

Comment:
I just noticed these, `ConfigUtils` should use a jicoco package.

Reply:
d'oh, good catch. Will fix.